controllers/apply/standard-proposal/constants.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const EXPIRY_EMAIL_REMINDERS = [
```

Comment:
Just wondering, given these are the same for awards for all do we need to define them twice? Could we pass an "enableExpiryReminders" flag to a form model which is what determines if we should set these? Definitely don't need to decide in this PR.

Reply:
The `emailType` keys are different (`STANDARD_ONE_WEEK` vs `AFA_ONE_WEEK`), although this was because I envisioned there being different email templates for each one of these, whereas we're actually sending the same email for all six types (just with some different variables). I'd like to keep differentiating them (if only for the database/logs so we know what types of email we've sent) but maybe there's a smart way to avoid duplicating the actual intervals/dates etc.

Reply:
Makes sense. This is probably one of those cases where duplicating until we have a clearer model for the data makes sense. Go with it as is 👍